### ISSUE TITLE
Preserve function argument comments

### ISSUE THREAD

**Issue by RobertVillalba** [gabotechs/graphqxl#57]
It seems that functions (queries/mutations) with comments for their arguments get converted into functions with the right arguments but no comments.
```
type Query {
  "Get all the things"
  getThings(
    "Some filter"
    filter: String
  ): [Thing!]!
}
```
becomes
```
type Query {
  "Get all the things"
  getThings(filter: String): [Thing!]!
}
```
Could we preserve the argument documentation?

**Comment by gabotechs** [comment]
yeah, this totally makes sense, and it's actually very easy to implement, I'll prepare a quick PR now

**Comment by gabotechs** [comment]
Released in v0.40.2, feel free to reopen if you find any issue

**Comment by RobertVillalba** [comment]
Awesome I can verify this works! Thank you!
